content/renderer/media/media_permission_dispatcher.cc

```diff
@@ -150,7 +154,8 @@ void MediaPermissionDispatcher::OnPermissionStatus(
   requests_.erase(iter);
 
 #if BUILDFLAG(USE_STARBOARD_MEDIA)
-  std::move(permission_status_cb).Run(true);
+  // DRM is only supported using StarboardRenderer.
+  std::move(permission_status_cb).Run(media::IsStarboardRendererEnabled());
```

Comment:
Just wanted to double check, do we want to always pass false here when StarboardRenderer is disabled, or shall we use the same logic as the one used below?